static/js/admin.js

```diff
@@ -657,6 +657,18 @@ function _setup_page() {
                         ui.report_success(i18n.t("waiting period threshold changed!"), waiting_period_threshold_status);
                     }
                 }
+                // Check if no changes made
+                if (response_data.name === undefined &&
+                    response_data.restricted_to_domain === undefined &&
+                    response_data.invite_required === undefined &&
+                    response_data.invite_by_admins_only === undefined &&
+                    response_data.create_stream_by_admins_only === undefined &&
+                    response_data.default_language === undefined &&
+                    response_data.add_emoji_by_admins_only === undefined &&
+                    response_data.allow_message_editing === undefined &&
+                    response_data.waiting_period_threshold === undefined) {
+                    ui.report_success(i18n.t("No changes were made!"), name_status);
```

Comment:
Is there a way we can make this work without needing to enumerate all the cases?

It seems like this is going to be fragile...